roles/redis/tasks/main.yml

```diff
@@ -28,9 +28,64 @@
     _node_affinity: "{{ raw_spec['affinity']['node_affinity'] | default({}) }}"
   when: affinity is defined and affinity.node_affinity is defined
 
+- name: Get the current redis pod information.
+  k8s_info:
+    api_version: v1
+    kind: Pod
+    namespace: '{{ ansible_operator_meta.namespace }}'
+    label_selectors:
+      - "app.kubernetes.io/name=redis"
+      - "app.kubernetes.io/instance=redis-{{ ansible_operator_meta.name }}"
+      - "app.kubernetes.io/component: cache"
```

Comment:
```suggestion
      - "app.kubernetes.io/component=cache"
```

Reply:
Thanks!  I pushed a fix for the broken label_selector.